### ISSUE TITLE
cannot verify

### ISSUE THREAD

**Issue by git12345abcd** [DediProgSW/SF100Linux#92]
When I use the SF100Linux to program the BIOS but the verification is still error.
Can you give some advices?

**Comment by Benwang217** [comment]
May I know the instructions and chip part number?

**Comment by git12345abcd** [comment]
Thank you,the chip is GD25LB512ME 
and the detail is as below:
When I use  sudo dpcmd -v to verify the program,but it is always failed.
When I use sudo dpcmd -p to program the roms or binary files,it is always success.
And now, I start the mainboard with the display,but the display is always shut down.

**Comment by Benwang217** [comment]
Hi,

-v can not be used alone, it should be used with -p
For example:
./dpcmd -p xxxx.bin -v

**Comment by git12345abcd** [comment]
Therefore,
as long as I program the binary or rom without -p only with -v is illegal?

**Comment by git12345abcd** [comment]
> Hi,
> 
> -v can not be used alone, it should be used with -p For example: ./dpcmd -p xxxx.bin -v

Therefore,
as long as I program the binary or rom without -p only with -v is illegal?

**Comment by Benwang217** [comment]
> > Hi,
> > -v can not be used alone, it should be used with -p For example: ./dpcmd -p xxxx.bin -v
> 
> Therefore, as long as I program the binary or rom without -p only with -v is illegal?

Yes, it is illegal. You can not use -v only, it should be used with -p.

**Comment by Benwang217** [comment]
If you would like to use -v only, you can use --loadFile-with-verify
